[FIX] Don't force arbitrary precision on weights

clonedagain opened the pull request:
The product view forces the decimal accuracy to 3 digits on weight, weight_net and volume, but the accuracy can actually be set using the 'Decimal Accuracy' feature so it must not be forced here

PR upstream: https://github.com/odoo/odoo/pull/6812

pedrobaeza commented:
:+1:

StefanRijnhart commented:
Not sure about this change. Weight and weight_net are governed by 'Stock Weight', which is set to 2, not 3. This could lead to rounding problems where 3 decimals are used. Volume does not have a precision in the field defintion. Not sure what the default is.

clonedagain commented:
You're right, I thought not having an accuracy set at the field definition level would cause it to default to 2 but it seems to accept 3 digits (so probably arbitrary precision is allowed and no rounding is done).
So I'll leave the volume alone.

Commit:
[FIX] Don't force arbitrary precision on weights

The product view forces the decimal accuracy to 3 digits on weight and weight_net, but the accuracy can actually be set using the 'Decimal Acuracy' feature so it must not be forced here

clonedagain commented:
@StefanRijnhart branch changed as you suggested.

StefanRijnhart commented:
Thanks! :+1:

clonedagain commented:
Merged upstream
